dataclients/kubernetes/kube.go

```diff
@@ -733,31 +733,41 @@ func catchAllRoutes(routes []*eskip.Route) bool {
 //
 // where for a weight of 1.0 no Traffic predicate will be generated.
 func computeBackendWeights(backendWeights map[string]float64, rule *rule) {
-	type sumCount struct {
-		sum   float64
-		count int
+	type pathInfo struct {
+		sum        float64
+		lastActive *backend
```

Comment:
I think you don't need this lastActive to achieve the goal of having the last set to 1.0

Reply:
Any suggestions, or at least a comment on why you don't like this approach?